layer_grid/SoftwareScreen.qml

```diff
@@ -247,7 +248,7 @@ FocusScope
                         
                         asynchronous: true
                         //smooth: true
-                        source: modelData.assets.screenshots[0] ? modelData.assets.screenshots[0] : ""
+                        source: modelData.assets.background
```

Comment:
I get what you're going for, but as someone who doesn't have fanart set for all my games, but has screenshots, I prefer screenshots. Maybe I can do an option in the future to toggle which to display, backgrounds instead screenshots

Reply:
No stress! I've come from gameOS which pulls background & logo for everything 👍

Reply:
Right, I'll probably look at gameOS when I go to add that functionality. But it has a preference for the backgrounds depending on the context that might be improved if it's a user toggle-able. Plus, need an excuse to get the buttons between the platformbar and controller help section working. Maybe one of those buttons will be for a Settings screen